packages-next/keystone/src/lib/server/createApolloServer.ts

```diff
@@ -62,16 +62,26 @@ const _createApolloServerConfig = ({
   graphQLSchema: GraphQLSchema;
   apolloConfig?: Config;
 }) => {
-  // Playground config
+  // Playground config, is /api/graphql available?
   const pp = apolloConfig?.playground;
   let playground: Config['playground'];
   const settings = { 'request.credentials': 'same-origin' };
+
+  // graphql.apolloConfig.playground === false (playground not accessible in all cases)
```

Comment:
Could you move all the comments so they live inside the `if() {}` block that they correspond to, rather than above the `if` statement. It's a little confusing in the current state.

Reply:
Done.